### ISSUE TITLE
Incorrect role is provided tags, icons and Project options as text: A11y_.NET 9 template_.NET 9 template_Project_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26817]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG4.1.2; #SH_.NET9template_MAC_Dec2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Name Role Value; #MAC;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
MAC Version 15.2

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till project section, and press enter on any project, Project screen will open,
4. Tab till tags.
5. Observe that Incorrect role is provided for tags, icons and Project options as text.

### Actual Result:
Incorrect role is provided for tags, icons and Project options as text.

### Expected Result:
Appropriate role should be provided for tags, icons and Project options.

### User Impact:
Screen reader users will get the confuse if incorrect role is provided for buttons available.

### Attachment:
https://github.com/user-attachments/assets/63981e67-e1ff-4628-87cb-a88eadf7d72d

<img width="1131" alt="Image" src="https://github.com/user-attachments/assets/fcfed8b9-2a24-4806-89d2-5e83c54f1953" />
<img width="1131" alt="Image" src="https://github.com/user-attachments/assets/d074923d-4ab0-436d-8012-5ec98999f0d5" />
<img width="1131" alt="Image" src="https://github.com/user-attachments/assets/c79ce6c8-369a-45bc-bacb-00b8f5ace863" />
<img width="1131" alt="Image" src="https://github.com/user-attachments/assets/cd32a7f2-80ef-49f2-9add-a5836cb0a4cf" />
<img width="1131" alt="Image" src="https://github.com/user-attachments/assets/1766af7e-8fe0-4a82-b343-3cfab498928b" />

**Comment by InduPriya1805** [comment]
Rev:Inse

**Comment by tj-devel709** [comment]
Note to self, the icons and the buttons below them are fixed, but the project selection cards are not yet fixed but will be when this one is fixed: https://github.com/dotnet/maui/issues/26815

**Comment by AnKushSingh05** [comment]
Issue is fixed

<img width="1105" alt="Image" src="https://github.com/user-attachments/assets/29815f24-5e50-4ec6-b307-fd090f219a72" />
<img width="1105" alt="Image" src="https://github.com/user-attachments/assets/62f019f5-afc1-450d-b51b-16e2c93f50f4" />
<img width="1105" alt="Image" src="https://github.com/user-attachments/assets/60bf8871-8621-4478-9faa-e08d95f9235a" />
